packages/flutter_tools/test/general.shard/artifacts_test.dart

```diff
@@ -89,7 +89,7 @@ void main() {
           .createSync(recursive: true);
       fileSystem
           .directory(xcframeworkPath)
-          .childDirectory('ios-armv7_arm64')
+          .childDirectory('ios-arm64_armv7')
```

Comment:
The changes to this file aren't strictly related to the codesign issue, but may as well change them together.